build/Dockerfile

```diff
@@ -1,12 +1,23 @@
+FROM golang:1.12 AS builder
+WORKDIR /src/vm-import-operator/
+COPY . .
+ENV GOFLAGS=-mod=vendor
+ENV GO111MODULE=on
+RUN CGO_ENABLED=0 GOOS=linux go build -o /csv-generator github.com/kubevirt/vm-import-operator/tools/csv-generator
+
 FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
 
 ENV OPERATOR=/usr/local/bin/vm-import-operator \
     USER_UID=1001 \
     USER_NAME=vm-import-operator
 
+ENV CSV_GENERATOR=/usr/bin/csv-generator \
+    USER_UID=1001 \
```

Comment:
Any reason to define USER_UID and USER_NAME twice? It's defined on line 11 & 12

Reply:
nope, squashed to a single ENV statement